### ISSUE TITLE
fix (test): improve error handling when calling contract with missing parameter

### ISSUE THREAD

**Issue by Daanvdplas** [r0gue-io/pop-node#120]
In the integration tests if we are calling a contract with a missing parameter (or incorrectly encoded probably as well) it will revert but there it no clear error returned. Would be good if this can be improved for future integration tests.

**Comment by chungquantin** [comment]
To provide more context, with out the data field (even empty vec<u8>), the contract call will be reverted:
```diff
fn transfer_from(
	addr: AccountId32,
	asset_id: AssetId,
	delegate: AccountId32,
	from: AccountId32,
	to: AccountId32,
	value: Balance,
+	data: Vec<u8>,
) -> ExecReturnValue {
	let function = function_selector("transfer_from");
	let params =
                 [function, asset_id.encode(), from.encode(), to.encode(), value.encode()]
+		[function, asset_id.encode(), from.encode(), to.encode(), value.encode(), data.encode()]
			.concat();
	let result = bare_call_by(addr, delegate, params, 0).expect("should work");
	result
}
```

**Comment by Daanvdplas** [comment]
Closed by #133
